src/condor_includes/sock.h

```diff
@@ -239,6 +239,8 @@ class Sock : public Stream {
         // RETURNS: true -- success; false -- failure
         //------------------------------------------
 
+        bool zkm_wrap(const unsigned char* input, int input_len,
+                  unsigned char*& output, int& outputlen);
```

Comment:
Please remove `zkm_*` variants and hide the side-effects (additional logging) behind a `#ifdef DEBUG` or similar.